src/main/java/frc/robot/commands/ManualElevatorControl.java

```diff
@@ -0,0 +1,36 @@
+//TODO: Need to figure out a way to ovverride the constant PID adjustment happening in Elevator.java subsystem to run this command
+
+
+package frc.robot.commands;
+
+import frc.robot.controls.Operator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class ManualElevatorControl extends CommandBase {
+  Elevator m_elevator; 
+
+  public ManualElevatorControl(Elevator elevator) {
+    m_elevator = elevator; 
+    addRequirements(m_elevator);
+  }
+  @Override
+  public void initialize() {
+    m_elevator.disablePID(); 
+  }
+  @Override
+  public void execute() {
+    m_elevator.set(Operator.getRawThrottleValue()); // rename throttlevalue function to somehting else
+    m_elevator.stopMotorsIfLimitSwitchesTripped(Operator.getRawThrottleValue());
+  }
+
+  @Override
+  public void end(boolean interrupted) {
```

Comment:
If it ends, it should re enable the PID.

Reply:
it should not end, I'll delete end() and finished() rn